### ISSUE TITLE
Module already loaded PHP7.1 only problem

### ISSUE THREAD

**Issue by jenstornell** [getkirby-v2/panel#977]
When I use PHP5.6 or PHP7.0 it works fine but when I use PHP7.1 I got this:

>Module 'xmlwriter' already loaded

If I disable it on my hosting it jumps to the next module...

>Module 'Phar' already loaded

![image](https://cloud.githubusercontent.com/assets/927011/21317117/06c0c520-c604-11e6-93bc-e9a30bebb364.png)

Because it only affects PHP7.1 it might be a bug on the Kirby side not compatible.

The rest of Kirby works fine, just the panel that has problems.

https://forum.getkirby.com/t/strange-new-error-module-xmlwriter-already-loaded/6093/1

**Comment by lukasbestle** [comment]
My site is running on PHP 7.1.0 since release and is working fine. To make sure, I have tried using the Panel again. Logging in and browsing around works. Do you get the errors on a specific page of the Panel? And can you verify it with a clean installation of the starterkit?

**Comment by jenstornell** [comment]
I tested a clean Starterkit installation with the exact same result. When I switched back to PHP7.0 it worked without problems.

I don't know if it helps but I use cPanel with these settings (unchanged and set by default).

![image](https://cloud.githubusercontent.com/assets/927011/21318118/3461577a-c608-11e6-81c8-dac5ce51d7c0.png)

The only difference is that in this screenshot I've swiched back to 7.0. Else it looks the same.

**Comment by lukasbestle** [comment]
Nope, we don't. We also don't set php.ini values except error reporting, sendmail and session settings.

**Comment by jenstornell** [comment]
I don't see how I can provide more information about this problem. I can live with PHP7.0 for a few years (I think), so it's fine for me at this time.

**Comment by lukasbestle** [comment]
I will keep an eye on it, maybe we can find the origin of the issue then.

**Comment by petrpscolka** [comment]
Hi, I have the similar problem. "Module 'dom' already loaded".
With Kirby 2.4.1, PHP 7.0 or with PHP 5.6.
Only with the old Kirby 2.2.3 is everything ok.

![bildschirmfoto 2017-01-09 um 14 44 24](https://cloud.githubusercontent.com/assets/3718349/21768785/bf5d4bee-d67b-11e6-825f-0c189d3d9b14.png)

**Comment by lukasbestle** [comment]
Interesting that it works with Kirby 2.2.3. Thanks for reporting!

**Comment by petrpscolka** [comment]
new info: 
it the url has direct "/de" or "/en" everithong is ok. it is only if i wrote the www.myurl.com – without the language slug.

**Comment by lukasbestle** [comment]
There is now an issue also related to modules but [exactly the other way around](https://forum.getkirby.com/t/php-startup-unable-to-load-dynamic-library/6572?u=lukasbestle). Very strange that it only happens with recent Kirby versions...

**Comment by bastianallgeier** [comment]
Would it be possible to get access to the server and run some tests directly there? Contact me via email if possible: bastian@getkirby.com

**Comment by jenstornell** [comment]
For me it's hard. I'm on a shared hosting but I can change the PHP version myself, but when I do, I do it for all my domains at the same time. I can't have longer downtimes, because I need income every day to survive.

However I tried to turn on PHP7.1 and log in to the panel now and today that worked. Maybe there is some upgrade done on my server and that this is a common issue when a server is moving from like 7.0 to 7.1. I'm not a server person but I can't reproduce the error anymore and I've tried it on 3 domains.

So for me this issue is solved for now. I'll let you know if I get in more trouble. It seems like it was never Kirby at all.
